core/server/common/src/main/java/alluxio/master/AlluxioVersionServiceHandler.java

```diff
@@ -1,41 +1,41 @@
 /*
- * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0 (the
- * "License"). You may not use this work except in compliance with the License, which is available
- * at www.apache.org/licenses/LICENSE-2.0
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
```

Comment:
nit: revert

Reply:
This file was not getting license check so I copied this from some other file.